Fastfile

```diff
@@ -117,6 +126,12 @@ default_platform :ios
     if is_ci && ENV['TAB_XCODE_PATH'] != nil
       xcode_select(ENV['TAB_XCODE_PATH'])
     end
+    if ENV['TAB_UI_TEST_SCHEME'] != nil && ENV['TAB_OUTPUT_TYPES'] == nil
```

Comment:
This feels like it belongs in setup not deploy to hockey

Reply:
@Neil3079 it is in setup isn't it?

Reply:
It is my bad!!!